pkg/rpcclient/unwrap/unwrap.go

```diff
@@ -180,6 +180,53 @@ func SessionIterator(r *result.Invoke, err error) (uuid.UUID, result.Iterator, e
 	return r.Session, iter, nil
 }
 
+// ArrayAndSessionIterator expects correct execution (HALT state) with one or two stack
+// items returned. If there is 1 item, it must be an array. If there is a second item,
+// it must be an iterator. This is exactly the result of smartcontract.CreateCallAndPrefetchIteratorScript.
+// NeoGo server SessionsEnabled parameter must be set to true.
+func ArrayAndSessionIterator(r *result.Invoke, err error) (uuid.UUID, result.Iterator, error) {
+	if err != nil {
```

Comment:
`Item` and friends check that the stack has exactly one item, reusing them seems clumsy, thus copying.

Reply:
Hey, there is `checkResOK()` for you!

Reply:
Oh, exactly, my first implementation didn't use this because the code wasn't here